cmd/cloud-controller-manager/app/controllermanager.go

```diff
@@ -184,7 +184,7 @@ func StartControllers(s *options.CloudControllerManagerServer, kubeconfig *restc
 		cloud.Initialize(clientBuilder)
 	}
 
-	versionedClient := client("shared-informers")
+	versionedClient := rootClientBuilder.ClientOrDie("shared-informers")
```

Comment:
Why don't we want shared-informed client to use ServiceAccount for authentication?

Reply:
I would have to regenerate with the old client to get the exact details. The client did not have the permissions on one of the objects it was watching.